http/server.ts

```diff
@@ -604,13 +603,15 @@ export async function serve(
   handler: Handler,
   options: ServeInit = {},
 ): Promise<void> {
-  const addr = options.addr ?? ":8000";
+  const addr = options.addr ?? "127.0.0.1:8000";
   const server = new Server({ addr, handler });
 
   if (options?.signal) {
     options.signal.onabort = () => server.close();
   }
 
+  console.log(`Listening on http://${addr}/`);
```

Comment:
This should probably use the resolved address, rather than the original address.